README.md

```diff
@@ -24,9 +26,32 @@ To compile the binary, run:
 go build main.go
 ```
 
-### Customizing
+### Docker
+
+To build the container after initial setup or making changes, run:
+
+```cli
+docker compose build
+```
+
+To run the binary from the container, run:
+
+```cli
+docker compose up
+```
+
+## Customizing
 
 Looking to shake things up? You can customize the emoji and the channels they should post to in `reaction-bot.go` by modifying `registeredReactions`. For configuration options on registering emoji, see the [docs][reaction_bot_register_emoji].
 
 [reaction_bot_repo]: http://github.com/jordanleven/reaction-bot
 [reaction_bot_register_emoji]: https://github.com/jordanleven/reaction-bot#registering-emoji
+
+## Deploying
+
+When changes are pushed to `main`, an image is built and published to the GitHub Container Registry. From the server, you can pull the image and restart the bot with following commands:
+
+```cli
+docker pull ghcr.io/sparkbox/reaction-bot
+docker compose up -d
```

Comment:
```suggestion
docker compose -f docker-compose.prod.yml up -d
```
Should we be specifying the prod configuration here?

Reply:
My assumption was that we'll just have a `.env` file and `docker-compose.yml` that matches the prod config in the repo. I think that's more or less what we're doing with sparkpress.